prow/docs/pr_status_setup.md

```diff
@@ -72,3 +74,7 @@ When testing locally, pass the path to your secrets to `deck` using the `--githu
 Run the commands:
 
 `go build . && ./deck --config-path=../../config.yaml --github-oauth-config-file=<PATH_TO_YOUR_GITHUB_OAUTH_SECRET> --cookie-secret=<PATH_TO_YOUR_COOKIE_SECRET> --oauth-url=/pr`
+
+## Run PR Status endpoint on a test cluster
+If hosting your instance on http instead of https, you will need to set `Secure` to `false` every time it appears in githuboauth/githuboauth.go.  
```

Comment:
You can link to the file like this:
```suggestion
If hosting your instance on http instead of https, you will need to set `Secure` to `false` every time it appears in [`githuboauth.go`](/prow/githuboauth/githuboauth.go).  
```

Please also link to the issue I just opened about making this configurable with a flag: https://github.com/kubernetes/test-infra/issues/12989